src/templates/registration_request_auto.html

```diff
@@ -0,0 +1,3 @@
+<span class="spinner login-submit"/>
+<p class="info">{{{info_message}}}</p>
```

Comment:
It doesn't look to me like you need a separate template. Instead you can simply use the existing `registration_request.html` template and add an "if" statement around the button which you don't want to show.